### PR TITLE
Use `ActivationStrategy` instead of `actionLevel` when available

### LINKED ISSUE #369: Handle deprecation of the `actionLevel` argument in finger crossed strategies
Form Symfony 6, the `actionLevel` must be an inner strategy instead of plain old value / string / int.

To prevent the deprecation, when creating the definition of the bridge's strategy, we should probably wrap the action level in an inner strategy as is done in the bridges ?

See https://github.com/symfony/monolog-bridge/blob/5.x/Handler/FingersCrossed/HttpCodeActivationStrategy.php#L39-L44 and https://github.com/symfony/monolog-bridge/blob/5.x/Handler/FingersCrossed/NotFoundActivationStrategy.php#L39-L44

### PR DESCRIPTION
**Pull request by jderusse**
Take over #336

Fixes #369

### CODE REVIEW

File: Tests/DependencyInjection/FixtureMonologExtensionTest.php [hunk 1]
```diff
@@ -11,15 +11,36 @@
...
 use Symfony\Component\DependencyInjection\Definition;
 use Symfony\Component\DependencyInjection\Reference;
 
 abstract class FixtureMonologExtensionTest extends DependencyInjectionTest
 {
+    /** @group legacy */
+    public function testLegacyLoadWithSeveralHandlers()
+    {
+        if (class_exists(SwitchUserTokenProcessor::class)) {
+            $this->markTestSkipped('Symfony Monolog < 5.2 is needed.');
```
Comment (derrabus) [inline Tests/DependencyInjection/FixtureMonologExtensionTest.php:28]:
```suggestion
            $this->markTestSkipped('Symfony MonologBridge < 5.2 is needed.');
```

**Review by derrabus — APPROVED** [review]

File: Tests/DependencyInjection/FixtureMonologExtensionTest.php (hunk 1 above)
Comment (derrabus) [reply Tests/DependencyInjection/FixtureMonologExtensionTest.php]: There are more places like this one. 😃

**Review by maxhelias — APPROVED** [review]

**Review by Jean85 — APPROVED** [review]

**Review by stof — APPROVED** [review]

**Comment by stephanvierkant** [comment]
Thanks! Any idea when a new tagged version will be released?

**Comment by jokaorgua** [comment]
@jderusse @stof would be really nice if you could give us ETA for a new tag with these fixes

**Comment by jokaorgua** [comment]
@jderusse @stof any ETA?

**Comment by patrykmartynowicz** [comment]
@jokaorgua As I see it's available already at `v3.7.0`.

**Comment by jokaorgua** [comment]
@patrykmartynowicz thanks. but due to some reason, GitHub marks 3.6.0 as the latest release

**Comment by derrabus** [comment]
> GitHub marks 3.6.0 as the latest release

Fixed. 😃

**Pull request merged**